.github/workflows/internal.yml

```diff
@@ -0,0 +1,33 @@
+name: Internal-unit-tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1
+    - name: Build Internal
+      working-directory: internal
```

Comment:
is the idea that once we get the `cmd` tests working they would run as a separate workflow?

Reply:
yup 👍 . Or we can change it to do a global `gotest -v ./...` to run all the unit tests in the same job, once the `cmd` ones are fixed